action/systemlog.go

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"math/big"
+
+	"github.com/golang/protobuf/proto"
+
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+
+	"github.com/iotexproject/iotex-core/pkg/util/byteutil"
+)
+
+// system log definitions
+type (
+	// TokenTxRecord is a token transaction record
+	TokenTxRecord struct {
```

Comment:
these 3 corresponds to 
https://github.com/iotexproject/iotex-proto/blob/master/proto/types/action.proto#L306-L320

Reply:
I probably put these three types in the wrong place in `iotex-proto`. I think these three are not actions. They are data holder which used on API we present to outside.